source/hands_on/gtrxl.rst

```diff
@@ -0,0 +1,139 @@
+GTrXL
+^^^^^^^
+
+Overview
+---------
+Gated Transformer-XL, or GTrXL, first proposed in `Stabilizing Transformers for Reinforcement Learning <https://arxiv.org/pdf/1910.06764.pdf>`_,
+is a novel framework for reinforcement learning based on the Transformer-XL architecture.
+It introduces several architectural modifications that improve the stability and learning speed of Transformer including:
+placing the layer normalization on only the input stream of the submodules, and replacing residual connections with gating layers.
+The proposed architecture, surpasses LSTMs on challenging memory environments and achieves state-of-the-art
+results on the several memory benchmarks, exceeding the performance of an external memory architecture.
+
+Quick Facts
+-------------
+1. GTrXL can serve as a **backbone** for many RL algorithms.
+
+2. GTrXL only supports **sequential** observations.
+
+3. GTrXL is based on **Transformer-XL** with **Gating connections**.
+
+4. The DI-engine implementation of GTrXL is based on the R2D2 algorithm. In the original paper, it is based on the algorithm V-MPO.
+
+Key Equations or Key Graphs
+---------------------------
+**Transformer-XL**: is a Transformer architecture that introduces the notion of recurrence to the deep self-attention network.
```

Comment:
To address the context fragmentation problem, Transformer-XL introduces the notion of recurrence to the deep self-attention network.

Reply:
added